### PR TITLE
[Raptor-3829] H2O Driverless AI Mojo Suport

### PR DESCRIPTION
**Pull request by timsetsfire**
update java_predictor.py and H2OPipelinePredictor

update readme for model_template/inference/h2o_mojo_pipeline

fix exception message in java_predictor.py

update readme 

# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

adding support for driverless ai pipeline exports as mojo.  This will not be included in automated testing and will require valid ongoing dai license when using dai mojos with drum.

### CODE REVIEW

File: custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py [hunk 1]
```diff
@@ -50,15 +53,19 @@ def __init__(self):
...
-            r"(\{})|(\{})|(\{})".format(*JavaArtifacts.ALL),
-            ",".join(os.listdir(self.custom_model_path)),
-        )
-        if ext_re is None:
-            files_list = sorted(os.listdir(self.custom_model_path))
-            files_list_str = " | ".join(files_list)
+        files_list = sorted(os.listdir(self.custom_model_path))
+        files_list_str = " | ".join(files_list)
+        self.logger.debug("files in custom model path: ".format(files_list_str))
+        reg_exp = r"(\{})|(\{})|(\{})|(\{})".format(*JavaArtifacts.ALL)
```
Comment (meltzg) [inline custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py:63]:
```suggestion
        reg_expr = r"|".join(r"(\{})".format(ext) for ext in JavaArtifacts.ALL)
```

**Review by meltzg — APPROVED** [review]
minor change to make code a bit more future proof, but otherwise LGTM

File: custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/scala/com/datarobot/custom/H2OPredictorPipeline.scala [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import scala.util.{Try, Success, Failure}
+
+import collection.JavaConverters._
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+class H2OPredictorPipeline(name: String) extends BasePredictor(name) {
+
+  var mojoPipeline: MojoPipeline = null
+  // var model: EasyPredictModelWrapper = null
```
Comment (yakov-g) [inline custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/scala/com/datarobot/custom/H2OPredictorPipeline.scala:20]: remove commented out

File: custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/scala/com/datarobot/custom/H2OPredictorPipeline.scala [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  var negativeClassLabel: String = null
+  var positiveClassLabel: String = null
+
+  def configure(
+      params: java.util.Map[String, Any] = new java.util.HashMap[String, Any]()
+  ) = {
+    mojoPipeline = loadModel(
+      params.get("__custom_model_path__").asInstanceOf[String]
+    )
+    // this.params = params
```
Comment (yakov-g) [inline custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/scala/com/datarobot/custom/H2OPredictorPipeline.scala:34]: and here

**Review by yakov-g — APPROVED** [review]

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**3.59** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**